mypy.ini

```diff
@@ -231,6 +231,12 @@ ignore_errors = True
 [mypy-torch.utils.tensorboard._caffe2_graph]
 ignore_errors = True
 
+[mypy-torch.utils.tensorboard.writer]
+ignore_errors = True
+
+[mypy-torch.utils.tensorboard._pytorch_graph]
+ignore_errors = True
```

Comment:
Can you resolve these, or are there a lot of new errors here? Re-adding new ignores for files that were already clean seems undesirable.

Reply:
Done! I forgot to regenerate the pyi files.